modules/spacecraftinstruments/rendering/renderablefov.cpp

```diff
@@ -69,69 +69,65 @@ namespace {
     };
 
-    constexpr openspace::properties::Property::PropertyInfo DefaultEndColorInfo = {
-        "Colors.DefaultEnd",
-        "End of default color",
-        "This value determines the color of the field of view frustum close to the "
-        "target. The final colors are interpolated between this value and the start "
-        "color.",
+    constexpr openspace::properties::Property::PropertyInfo ColorDefaultEndInfo = {
+        "DefaultEnd",
+        "Default End",
+        "The color to use for the field of view frustum close to the target. The final "
+        "colors are interpolated between this value and the start color.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo ActiveColorInfo = {
-        "Colors.Active",
-        "Active Color",
-        "This value determines the color that is used when the instrument's field of "
-        "view is active.",
+    constexpr openspace::properties::Property::PropertyInfo ColorActiveInfo = {
+        "Active",
+        "Active",
+        "The color that is used when the instrument's field of view is active.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo TargetInFovInfo = {
-        "Colors.TargetInFieldOfView",
-        "Target in field-of-view Color",
-        "This value determines the color that is used if the target is inside the field "
-        "of view of the instrument but the instrument is not yet active.",
+    constexpr openspace::properties::Property::PropertyInfo ColorTargetInFovInfo = {
+        "TargetInFieldOfView",
+        "Target in Field of View",
+        "The color that is used if the target is inside the field of view of the "
+        "instrument but the instrument is not yet active.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo IntersectionStartInfo = {
-        "Colors.IntersectionStart",
-        "Start of the intersection",
-        "This value determines the color that is used close to the instrument if one of "
-        "the field of view corners is intersecting the target object. The final color is "
-        "retrieved by interpolating between this color and the intersection end color.",
+    constexpr openspace::properties::Property::PropertyInfo ColorIntersectionStartInfo = {
+        "IntersectionStart",
+        "Intersection Start",
+        "The color that is used close to the instrument if one of the field of view "
+        "corners are intersecting the target object. The final color is is an "
+        "interpolation of this color and the intersection end color.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    constexpr openspace::properties::Property::PropertyInfo IntersectionEndInfo = {
-        "Colors.IntersectionEnd",
```

Comment:
I'm not sure how removing the subname impacts getting the values out of the dictionary?

Reply:
I added a property owner with the same name as the one here, so the URI to the properties should still be the same.

Note that we currently cannot specify them in the asset, though...

Reply:
I think we can fix that at a later time.  The RenderableFOV is pretty old and probably due for a do-over anyway